src/test/scala/uk/gov/hmrc/perftests/example/TrackingConsentFrontendRequests.scala

```diff
@@ -40,13 +40,24 @@ object TrackingConsentFrontendRequests extends ServicesConfiguration {
       .check(status.is(200))
   }
 
+  val hmrcFrontendVersion: String = {
```

Comment:
I would probably just inline this inside `getCookieSettingsPage`, as it doesn't need to be public